Document usage without pre-commit (#1)
gindex/recipemd-validator: README.md

    hooks:
      - id: validate-recipemd
```

### Without pre-commit

> If you want to use this package without pre-commit, install it using ``pip install "git+git://github.com/gindex/recipemd-validator"`` and put the following in your ``.git/hooks/pre-commit.sh``:

Reviewer: ```suggestion
If you want to use this package without pre-commit, install it using `pip install "git+https://github.com/gindex/recipemd-validator.git"` and put the following in your `.git/hooks/pre-commit` and make the hook executable `chmod +x .git/hooks/pre-commit`:
```

Looks like `git+git` is no longer supported, I'm getting `fatal: remote error: The unauthenticated git protocol on port 9418 is no longer supported.`. Let's use https here.
